res/controllers/Reloop-Jockey-3-ME-scripts.js

```diff
@@ -0,0 +1,444 @@
+function Jockey3ME() {}
+
+// Variables
+Jockey3ME.EffectLedMeter = 0;
+Jockey3ME.EffectLedMeterValue = 1;
+Jockey3ME.LedMeterShowValue = 1;
+Jockey3ME.LedMeterShowValueTwo = false;
+Jockey3ME.VuMeter = 0;
+Jockey3ME.scratching = [];
+Jockey3ME.hotcueClearVal = 0;
+Jockey3ME.crossfaderScratch = false;
+Jockey3ME.effectSelectTimer = 0;
+Jockey3ME.num_effectsValue = [0,0,0,0];
+Jockey3ME.effectsAvailable = 5; // Sets how many Effects are Loadable
+Jockey3ME.move_beat_value = 4; // Sets how many Beats Jumping when "MOVE" is Turned
+Jockey3ME.CUP_value = 0;
+Jockey3ME.MixerDeck1 = 0;
+Jockey3ME.MixerDeck2 = 0;
+Jockey3ME.noVolHopValue = [false,false,false,false,false];
+
+// Functions
+Jockey3ME.EffectLedMeterShow = function () {
+  midi.sendShortMsg(0x90,0x1D,Jockey3ME.EffectLedMeterValue);
+  midi.sendShortMsg(0x91,0x1D,Jockey3ME.EffectLedMeterValue);
+  Jockey3ME.EffectLedMeterValue += 2;
+  if (Jockey3ME.EffectLedMeterValue >= 127) {
+    engine.stopTimer(Jockey3ME.EffectLedMeter);
+    Jockey3ME.EffectLedMeter = 0;
+
+    // Sets Effect Leds
+    for (var i = 1, j = 176; i <= 4; i++) {
+      Jockey3ME.effectSelectLedSet(j,i);
+      j++;
+    }
+  };
+}
+// Main LedShow Script
+Jockey3ME.LedMeterShow = function() {
+  midi.sendShortMsg(0x90,0x21,Jockey3ME.LedMeterShowValue);
+  midi.sendShortMsg(0x91,0x21,Jockey3ME.LedMeterShowValue);
+  if (Jockey3ME.LedMeterShowValueTwo == false) {
+    ++Jockey3ME.LedMeterShowValue;
+  } else {
+    --Jockey3ME.LedMeterShowValue;
+  }
+  if (Jockey3ME.LedMeterShowValue > 10) Jockey3ME.LedMeterShowValueTwo = true;
+  // Stop The LedShow and Start Scanning VuMeter
+  if (Jockey3ME.LedMeterShowValueTwo && Jockey3ME.LedMeterShowValue <= 0) {
+    engine.stopTimer(Jockey3ME.LedMeterShowTimer);
+    Jockey3ME.LedMeterShowTimer = 0;
+    Jockey3ME.VuMeter = engine.beginTimer(20,"Jockey3ME.fVuMeter()"); // Start Every 20ms the fVuMeter Function
+    Jockey3ME.EffectLedMeter = engine.beginTimer(20,"Jockey3ME.EffectLedMeterShow()");
```

Comment:
Did you mean line 51 or 52?
I have looked to the wiki and read the midi file format page again, so now ill be clear that i don't need line 51 anymore.
My bad sry.